cmd/media.go

```diff
@@ -150,11 +162,25 @@ func handleDeleteMedia(c echo.Context) error {
 	}
 
 	app.media.Delete(fname)
-	app.media.Delete(thumbPrefix + fname)
+	app.media.Delete(thumbPrefix + fname + addPrefixIfNeeded(fname))
 
 	return c.JSON(http.StatusOK, okResp{true})
 }
 
+// generate a png thumbnail of src
+func getSVGThumb(src multipart.File) (image.Image, error) {
```

Comment:
I think we don't need this at all. SVG is vector, so for thumbnails, it can just be scaled down and rendered. The thumb path and the full path can be the same image.